src/yardstick/capture.py

```diff
@@ -81,12 +81,26 @@ def intake(config: artifact.ScanConfiguration, raw_results: str) -> artifact.Sca
     return artifact.ScanResult(config=config, metadata=metadata, **keys)
 
 
+def summarize_image(image: str):
```

Comment:
Are this method and `summarize_tool` below meant to be used somewhere? I didn't see any references, but maybe I missed something.

Reply:
yeah, the rebase got a little rough. I'll put it back

Reply:
I'll yank it entirely from the PR